doc/060_forget.rst

```diff
@@ -219,6 +219,7 @@ paths and tags. The policy is then applied to each group of snapshots individual
 This is a safety feature to prevent accidental removal of unrelated backup sets. To
 disable grouping and apply the policy to all snapshots regardless of their host,
 paths and tags, use ``--group-by ''`` (that is, an empty value to ``--group-by``).
+The ``--group-by`` option should also be set accordingly for the ``backup`` command.
```

Comment:
Honestly, I think that it is *normal* for these two to be set equally, but it's not *always* what you want to do. So maybe make this:
~~~
Note that one would normally set the ``--group-by`` option for the ``backup`` command to the same value.
~~~